src/js/utils/common.js

```diff
@@ -0,0 +1,67 @@
+export {
+  inputError,
+  onFocusoutEmailInput,
+  onFocusoutPasswordInput,
+  passwordToggle,
+  passwordCheckToggle,
+};
+
+import { EMAIL_REGEX } from "./constant.js";
+import {
+  emailInput,
+  errorMessage,
+  passwordInput,
+  passwordEyeOff,
+  passwordEyeOn,
+  passwordCheckEyeOff,
+  passwordCheckEyeOn,
+  passwordCheck,
+} from "./tags.js";
+
+const inputError = (input, element, message) => {
+  input.classList.add("error");
+  element.textContent = message;
+  element.classList.add("error-message");
+};
+
+const onFocusoutEmailInput = (e) => {
+  if (emailInput.value === "") {
+    inputError(emailInput, errorMessage, "이메일을 입력해 주세요.");
+  } else if (!EMAIL_REGEX.test(emailInput.value)) {
+    inputError(emailInput, errorMessage, "올바른 이메일 주소가 아닙니다.");
+  }
+  document.querySelector(".email-input-container").append(errorMessage);
+  return;
+};
+
+const onFocusoutPasswordInput = (e) => {
+  if (passwordInput.value === "") {
+    inputError(passwordInput, errorMessage, "비밀번호를 입력해 주세요.");
```

Comment:
@gaem-gaem 

[제안]

이벤트핸들러 콜백함수로 전달받은 매개변수를 활용해보시는건 어떨까요?

```suggestion
const onFocusoutPasswordInput = (e) => {
  const value = e.target.value
  if (value === "") {
...
```